### PR TITLE
Test order of focus-related events

### PR DESCRIPTION
**Pull request by cvrebert**

### CODE REVIEW

**Comment by cvrebert** [comment]
CC: @zcorpan as a potential reviewer

File: uievents/order-of-events/focus-events/focus-manual.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    for (var k = 0; k < relevantEvents.length; k++) {
+      inputs[i].addEventListener(relevantEvents[k], record, false);
+    }
+  }
+
+  async_test(function(t) {
+    document.getElementById('done').addEventListener('click', function () {
+      finish();
+      t.step(function () {
+        assert_equals(
```
Comment (zcorpan) [inline uievents/order-of-events/focus-events/focus-manual.html]: Use assert_array_equals

File: uievents/order-of-events/focus-events/focus-manual.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          events,
+          ['focusin', 'focus', 'focusout', 'focusin', 'blur', 'focus'],
+          'Focus-related events should fire in this order: focusin, focus, focusout, focusin, blur, focus'
+        );
+        assert_equals(
```
Comment (zcorpan) [inline uievents/order-of-events/focus-events/focus-manual.html]: Also here

File: uievents/order-of-events/focus-events/legacy-manual.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    for (var k = 0; k < relevantEvents.length; k++) {
+      inputs[i].addEventListener(relevantEvents[k], record, false);
+    }
+  }
+
+  async_test(function(t) {
+    document.getElementById('done').addEventListener('click', function () {
+      finish();
+      t.step(function () {
+        assert_equals(
```
Comment (zcorpan) [inline uievents/order-of-events/focus-events/legacy-manual.html]: also here

File: uievents/order-of-events/focus-events/legacy-manual.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+          events,
+          ['focusin', 'focus', 'DOMFocusIn', 'focusout', 'focusin', 'blur', 'DOMFocusOut', 'focus', 'DOMFocusIn'],
+          'Focus-related events should fire in this order: focusin, focus, DOMFocusIn, focusout, focusin, blur, DOMFocusOut, focus, DOMFocusIn'
+        );
+        assert_equals(
```
Comment (zcorpan) [inline uievents/order-of-events/focus-events/legacy-manual.html]: also here

**Comment by zcorpan** [comment]
LGTM with assert_array_equals

**Comment by cvrebert** [comment]
Switched to `assert_array_equals`.

**Pull request merged**